python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -199,3 +201,118 @@ def mef_filename(phosim_amp_file, out_dir='.'):
         if outfile.endswith('.gz'):
             outfile = outfile[:-len('.gz')]
         return outfile
+
+    def process_visit_eimg(self, visit_dir, out_dir=None, prefix='lsst',
```

Comment:
The method name is already pretty long.  I'd think about calling it `process_visit_eimage` just to be explicit.  Same suggestion for methods below.

Reply:
Corrects in the new commit: edbbf54.